.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -198,14 +198,16 @@ dynamodb_BatchWriteItem:
       versions:
         - sdk_version: 3
           github: javascriptv3/example_code/dynamodb
-          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-table-read-write-batch.html#dynamodb-example-table-read-write-batch-writing
           excerpts:
             - description: Create the client.
               snippet_tags:
-                - dynamodb.JavaScript.tables.createclientv3
+                - dynamodb.JavaScript.scenario.basics.createclientv3
+            - description: Create the Document client.
+              snippet_tags:
+                - dynamodb.JavaScript.scenario.basics.createdocclientv3
             - description: Add the items to the table.
               snippet_tags:
-                - dynamodb.JavaScript.batch.WriteItemV3
+                - dynamodb.JavaScript.scenario.basics.createdocclientv3

```

Comment:
Wrong snippet (this is the "create doc client" snippet again, instead of the "batch write items" snippet).

Reply:
done